buildscripts/builder.sh

```diff
@@ -1,173 +0,0 @@
-#!/bin/bash
-
-# INIT
-MACHINE_TYPE=`uname -m`
-BASE_DIR=$(pwd)
-PYTHON_VERSION=2.7.15
-PYQT_VERSION=4-4.11.4-gpl-Py2.7-Qt4.8.7
-OPENSSL_VERSION=1_0_2t
-DIRECTORY32BIT=SoftwareDownloads32bit
-DIRECTORY64BIT=SoftwareDownloads64bit
-
-if [ ${MACHINE_TYPE} == 'x86_64' ]; then
-	if [ ! -d "$DIRECTORY64BIT" ]; then
-		mkdir SoftwareDownloads64bit
-		cd SoftwareDownloads64bit
-	else
-		echo "Directory already exists"
-		cd SoftwareDownloads64bit
-	fi
-else
-	if [ ! -d "$DIRECTORY32BIT" ]; then
-		mkdir SoftwareDownloads32bit
-		cd SoftwareDownloads32bit
-	else
-		echo "Directory 32 bit alrready exists"
-		cd SoftwareDownloads32bit
-	fi
-fi
-#Functions
-function install_wine {
-
-	
-	wget -nc https://dl.winehq.org/wine-builds/Release.key  --no-check-certificate
-	sudo apt-key add Release.key
-	sudo apt-add-repository 'https://dl.winehq.org/wine-builds/ubuntu/'
-	sudo apt-get -y update
-	sudo apt-get -y install wine1.8 winetricks
```

Comment:
That apt commands are probably needed

Reply:
Maybe `sudo apt-get -y install wine1.8 winetricks` is sufficient

Reply:
line 145-148 show what needs to be installed.